scripts/HPC_chatbot_preprocessor/README.md

```diff
@@ -0,0 +1,29 @@
+# Chatbot parser
+
+`chatbot_parser.py` is a script that transforms the markdown sourcefiles into a structured directory for a chatbot to be trained on. 
```

Comment:
is there a way to force that this script runs on eg every commit?
it would be nice to see (in the PR) how any changes to the (real) md sources refelct in the bot sources.

and maybe replace the `for a chatbot to be trained on` with `as input for a chatbot`. it's very uncertain ifthis will ever be used for any training/finetuning

Reply:
> is there a way to force that this script runs on eg every commit?

Sure, we can set up a GitHub Actions workflow for this I think...

Not sure we can/should do that as a part of this PR though (since the workflow would need write access to the PR branch to update files)